README.md

```diff
@@ -193,6 +200,44 @@ You can force the direction of a relationship by using:
 * `Rel_L`, `Rel_Left`
 * `Rel_R`, `Rel_Right`
 
+Relationship specific sprites are not down scaled, they requires typically smaller icons.
+Therefore if sprite argument starts with `&` an OpenIconic name can be used too (details see https://useiconic.com/open)
+
+```csharp
+@startuml
+!include https://raw.githubusercontent.com/plantuml-stdlib/C4-PlantUML/master/C4_Dynamic.puml
+
+Person(user, "User1")
+Person(user2, "User2")
+System(system, "System")
+
+' if sprite starts with &, sprite defines a OpenIconic, details see https://useiconic.com/open/
```

Comment:
GitHub shows syntax errors here but I have no idea _why_. Mixed line-endings? Null characters? :confused: 

![image](https://user-images.githubusercontent.com/195757/111066287-78300e00-84be-11eb-8918-b59391aa21a5.png)

Reply:
do you mean the red color blocks?